MapleServer2/Packets/JobPacket.cs

```diff
@@ -69,5 +69,28 @@ public static Packet WriteSkills(PacketWriter pWriter, Player character)
 
             return pWriter;
         }
+
+        public static Packet WritePassiveSkills(PacketWriter pWriter, IFieldObject<Player> character)
+        {
+            // The x.Value.Learned == 1 is to filter for now, the skills by level 1 until player can be save on db.
+            List<SkillMetadata> passiveSkillList = character.Value.SkillTabs[0].SkillJob.Where(x => x.Value.Type == 1 && x.Value.Learned == 1).Select(x => x.Value).ToList();
```

Comment:
What are the different skill types? Is Type == 1 always a passive skill?

Reply:
Yes